src/core/ipfs/files.js

```diff
@@ -53,6 +53,7 @@ module.exports = function files (self) {
       pull(
         pull.values([hash]),
         pull.asyncMap(self._dagS.get.bind(self._dagS)),
+        pull.take(1),
```

Comment:
This is absurd. It's serious fancy abstraction creep. Careful

On Mon, Sep 12, 2016 at 10:53 AM Friedel Ziegelmayer <
notifications@github.com> wrote:

> In src/core/ipfs/files.js
> <https://github.com/ipfs/js-ipfs/pull/469#discussion_r78388308>:
>
> > @@ -53,6 +53,7 @@ module.exports = function files (self) {
> >        pull(
> >          pull.values([hash]),
> >          pull.asyncMap(self._dagS.get.bind(self._dagS)),
> > +        pull.take(1),
>
> sorry, nvm what I'm doing here is stupid -.-
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipfs/js-ipfs/pull/469/files/06ee44fb061954f440d6c80df94609400dc8f12e#r78388308>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIcoW2OqPNNILjFBhda_sNfyhs_gxARks5qpWd6gaJpZM4J5Uvl>
> .
>